simpleLooping/addTwoBinaryList.py

```diff
@@ -6,3 +6,32 @@
 # Ex:
 #   addTwoBinaryList([1, 0, 1], [1, 0, 0, 1]) # return [0, 1, 1, 1]
 #   addTwoBinaryList([1, 1, 0], [1, 0, 1, 1]) # return [0, 0, 0, 0, 1]
+from typing import List
+
+def addTwoBinaryList(binaryList1, binaryList2):
+    lenBinaryList = max(len(binaryList1), len(binaryList2))
+
+    binaryList1 = [0] * (lenBinaryList - len(binaryList1)) + binaryList1
+    binaryList2 = [0] * (lenBinaryList - len(binaryList2)) + binaryList2
+
+    binaryListTotal = [0] * (lenBinaryList + 1)
+
+    currentIndex = lenBinaryList
+    
+    for i in range(lenBinaryList-1, -1, -1):
```

Comment:
Why you are reverse order like this? I thought in normal addition, we got to add numA[i] + numB[i] from i = 0 to i = 1 and so on. 

Please don't just fix the code by making it become "convenience" you need to identify root cause and fix the root cause.